.github/workflows/release.yml

```diff
@@ -22,13 +22,14 @@ on:
         - beta
         - rc
         - stable
-      host:
-        description: "Which host package to release"
+      package:
```

Comment:
Can't comment on unchaged line, but I am thinking changing `all` to `all_hosts` or just `hosts, because I don't think there will be cases when we want to release parser, onesdk, ... with same release level and/or kind.

Reply:
Makes sense to me